src/lib/markbind/src/parser.js

```diff
@@ -27,107 +26,16 @@ cheerio.prototype.options.decodeEntities = false; // Don't escape HTML entities
 const {
   ATTRIB_INCLUDE_PATH,
   ATTRIB_CWF,
-  IMPORTED_VARIABLE_PREFIX,
 } = require('./constants');
 
 class Parser {
-  constructor(options) {
-    this._options = options || {};
-    this._fileCache = {};
```

Comment:
It seems like our new implementation no longer caches included files. Should we add this to the `variablePreprocessor`?